### ISSUE TITLE
Docker builds failing due to docker-17.09.0-ce.tgz checksum mismatch

### ISSUE THREAD

**Issue by edmorley** [travis-ci/travis-cookbooks#936]
Multiple variations of the `BUILDER=docker` packer-templates builds are failing using travis-cookbooks master, due to:

```
docker:       - create new directory /usr/local/docker-17.09.0-ce
docker:     * remote_file[/tmp/packer-chef-solo/local-mode-cache/cache/docker-17.09.0-ce.tgz] action create
docker:
docker:       ================================================================================
docker:       Error executing action `create` on resource 'remote_file[/tmp/packer-chef-solo/local-mode-cache/cache/docker-17.09.0-ce.tgz]'
docker:       ================================================================================
docker:
docker:       Chef::Exceptions::ChecksumMismatch
docker:       ----------------------------------
docker:       Checksum on resource (e9902c) does not match checksum on content (a9e90a)
```

For example:
https://travis-ci.org/travis-infrastructure/packer-build/jobs/298683570#L1834
https://travis-ci.org/travis-infrastructure/packer-build/jobs/298063694#L1608
https://travis-ci.org/travis-infrastructure/packer-build/jobs/299186924#L1553

The file in question:
https://download.docker.com/linux/static/stable/x86_64/docker-17.09.0-ce.tgz

The sha256 checksum I get for it now:
`a9e90a73c3cdfbf238f148e1ec0eaff5eb181f92f35bdd938fd7dab18e1c4647`

The checksum in the cookbook:
`e9902c16a81b67830bf361074e5ffc3333bf48a3e37e2ef0544fd9955bd3f1f7`

**Comment by edmorley** [comment]
I couldn't find anything in Docker issues trackers about the archive being updated, and looking at https://travis-ci.org/travis-infrastructure/packer-build/builds shows the last definitely green build as:
https://travis-ci.org/travis-infrastructure/packer-build/builds/289054274

As such I think the checksum might have just been wrong when it landed in #928, since there were no builds triggered there.

I'll open a PR to update it.

**Comment by soulshake** [comment]
Thank you @edmorley!
